server.go

```diff
@@ -62,6 +62,7 @@ func (s *server) Start() (bool, error) {
 	}, bufSize)
 
 	<-s.persister.ConfigChangedWatcher()
+	s.createBucketContainer()
```

Comment:
question: Do we need to call `createBucketContainer` after `<-s.persister.ConfigChangedWatcher()`? Or it won't matter?